node.go

```diff
@@ -342,7 +342,27 @@ func (n *Node) LoadCommandClassVersions() error {
 			return err
 		}
 
-		time.Sleep(interviewTimeout)
+		// Record time that we started waiting for a version report
+		waitingForVersionSince := time.Now()
+
+		// Wait until either the version has been reported, or we've waited too long
+		// 5 seconds is used to be sure any collisions (and thus CAN timeouts) are over
+		for {
+			// Check to see if this command class has been reported
+			if v := n.CommandClasses.GetVersion(commandClassID); v != 0 {
+				break
+			}
+
+			if waitingForVersionSince.Add(time.Second * 5).Before(time.Now()) {
```

Comment:
So I think my one major concern is that this timeout is too long. I think some devices might give up on pairing before we would get through each of the interviews.

Reply:
Good call. I'm going to decrease this to 3.5 seconds and test further